BoxOffice/NetWork/APIType.swift

```diff
@@ -1,41 +1,64 @@
 //
-//  KobisAPIType.swift
+//  APIType.swift
 //  BoxOffice
 //
 //  Created by karen on 2023/08/06.
 //
 
 import Foundation
 
-enum KobisAPIType {
+enum APIType {
     case movie(String)
     case boxOffice(String)
+    case movieImage(String)
 }
 
-extension KobisAPIType {
+extension APIType {
     var urlComponents: URLComponents? {
         var urlComponents = URLComponents()
+        
         urlComponents.scheme = scheme
         urlComponents.host = host
         urlComponents.path = path
         urlComponents.queryItems = queries
+        
         return urlComponents
     }
     
     var header: String? {
-      return nil
+        switch self {
+        case .movieImage(_):
```

Comment:
```suggestion
        case .movieImage:
```

연관값을 사용하지 않고자 한다면 괄호를 생략하는 방법도 있습니다 😄

Reply:
꼭 넣어줘야되는지 알았습니다. 이번에 새롭게 알게되었으니 해당 부분 적용시켜보도록 하겠습니다.
감사합니다. 븨븨~😄